<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_serverlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_join">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>加入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_brocast">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_brocast">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>广播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_message">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_users">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>160</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_messagearea">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
